components/core/CarouselSidebarData.js

```diff
@@ -191,26 +191,24 @@ const STYLES_AUTOSAVE = css`
   opacity: 0;
   margin: 26px 24px;
 
-  @keyframes autosave {
+  @keyframes slate-animations-autosave {
     0% {
-      height: 0;
       opacity: 0;
-      margin-left: 0;
+      transform: translateX(0);
     }
     10% {
       opacity: 1;
-      margin-left: 24px;
+      transform: translateX(12px);
     }
     90% {
       opacity: 1;
-      margin-left: 24px;
+      transform: translateX(12px);
     }
     100% {
       opacity: 0;
-      height: 0;
     }
   }
-  animation: autosave 4000ms ease;
+  animation: slate-animations-autosave 4000ms ease;
```

Comment:
This would be a nice cleanup task to consolidate all of these at some point, really nice though.